subgraph/flake.nix

```diff
@@ -0,0 +1,49 @@
+{
+  description = "Flake for development metaboard subgraph workflows.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/ec750fd01963ab6b20ee1f0cb488754e8036d89d";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+
+      in rec {
+        packages = rec {
+          init =  pkgs.writeShellScriptBin "init" (''
+            npm install
+            mkdir -p contracts && cp -r ../src/* contracts
+
+            npx hardhat compile --force
+
+            rm -rf ./abis && mkdir ./abis
+            cp artifacts/contracts/concrete/MetaBoard.sol/MetaBoard.json abis
+          '');
+
+          ci-test = pkgs.writeShellScriptBin "ci-test" (''
+            npx mustache config/localhost.json subgraph.template.yaml subgraph.yaml
```

Comment:
why are we using npx in nix?

Reply:
This is something @NanezX should look into probably

Reply:
Where did you get that? Currently we don't use npx or mustache for generating the YAML with the address. We use rain.cli for that

Reply:
@NanezX can you post the link to that please, thanks